apps/wallet-mobile/src/WalletNavigator.tsx

```diff
@@ -175,7 +180,7 @@ export const WalletNavigator = () => {
         detachPreviousScreen: false /* https://github.com/react-navigation/react-navigation/issues/9883 */,
       }}
     >
-      {showBiometricsScreen && authOsEnabled && (
+      {!hasWallets && showBiometricsScreen && isAuthOsSupported && authSetting !== 'os' && (
```

Comment:
so we don't have to compile on head.

```suggestion
const shouldAskToUseAuthWithOs = !hasWallets && showBiometricsScreen && isAuthOsSupported && authSetting !== 'os'

      {shouldAskToUseAuthWithOs && (
```